ProtoGame2D/Code/Game/Framework/App.cpp

```diff
@@ -36,27 +34,21 @@ void App::Startup()
 	g_Window->Init("ProtoGame2D");
 	g_Window->SetInputSystem(g_InputSystem);
 
+	LogStartup();
+	
 	Renderer::CreateInstance();
 	Renderer::GetInstance()->StartUp();
 
-	LogStartup();
-
-	m_ShaderDef = new ShaderDefinition(*ShaderDefinition::InitializeDef("Data/Materials/shader.xml"));
-	m_Shader = Renderer::GetInstance()->GetOrCreateShader(m_ShaderDef);
-
-	m_ScreenShaderDef = new ShaderDefinition(*ShaderDefinition::InitializeDef("Data/Materials/screenShader.xml"));
-	m_ScreenShader = Renderer::GetInstance()->GetOrCreateShader(m_ScreenShaderDef);
-
-	Renderer::GetInstance()->EnableBlend(ParseBlendFac[m_Shader->GetSRC()], ParseBlendFac[m_Shader->GetDST()], ParseBlendOp[m_Shader->GetOP()]);
+	m_Time = new Time(60);
```

Comment:
Remove seed from time and move to an RNG class for random number generation stuff

Reply:
done